src/test/java/com/exasol/rls/administration/scripts/DeleteRlsRoleIT.java

```diff
@@ -0,0 +1,139 @@
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideValuesForTestDeleteRlsRoleFromExaRlsUsers")
+    void testDeleteRlsRoleFromExaRlsUsers(final String roleToDelete, final String expectedTableContent)
+            throws SQLException {
+        ScriptsSqlManager.createExaRolesMappingProjection(statement, EXA_ROLES_MAPPING,
+                "('Sales', 1), ('Development', 2), ('Finance', 3),  ('Support', 4)");
+        ScriptsSqlManager.createExaRlsUsersProjection(statement, EXA_RLS_USERS, "('RLS_USR_1', 15), ('RLS_USR_2', 9)");
+        statement.execute("EXECUTE SCRIPT DELETE_RLS_ROLE(" + roleToDelete + ")");
+        ScriptsSqlManager.createExaRlsUsersProjection(statement, EXA_RLS_USERS_PROJECTION, expectedTableContent);
+        final ResultSet expectedResultSet = statement.executeQuery("SELECT * FROM " + EXA_RLS_USERS_PROJECTION);
+        final ResultSet actualResultSet = statement.executeQuery("SELECT * FROM " + EXA_RLS_USERS);
+        ScriptsSqlManager.dropTable(statement, EXA_ROLES_MAPPING);
+        ScriptsSqlManager.dropTable(statement, EXA_RLS_USERS);
+        ScriptsSqlManager.dropTable(statement, EXA_RLS_USERS_PROJECTION);
+        assertThat(actualResultSet, matchesResultSet(expectedResultSet));
+    }
+
+    private static Stream<Arguments> provideValuesForTestDeleteRlsRoleFromExaRlsUsers() {
+        return Stream.of(Arguments.of("'Sales'", "('RLS_USR_1', 14), ('RLS_USR_2', 8)"), //
+                Arguments.of("'Development'", "('RLS_USR_1', 13), ('RLS_USR_2', 9)"), //
+                Arguments.of("'Finance'", "('RLS_USR_1', 11), ('RLS_USR_2', 9)"), //
+                Arguments.of("'Support'", "('RLS_USR_1', 7), ('RLS_USR_2', 1)"));
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideValuesForTestDeleteRlsRoleFromUserTable")
+    void testDeleteRlsRoleFromUserTable(final String roleToDelete, final String expectedTableContent)
+            throws SQLException {
+        ScriptsSqlManager.createExaRolesMappingProjection(statement, EXA_ROLES_MAPPING,
+                "('Sales', 1), ('Development', 2), ('Finance', 3),  ('Support', 4)");
+        final String test_table_name = "TEST_TABLE";
+        createUserTable(test_table_name, "('Row1', 1), ('Row2', 6), ('Row3', 9),  ('Row4', 15)");
+        final String test_table_projection_name = "TEST_TABLE_PROJECTION";
+        createUserTable(test_table_projection_name, expectedTableContent);
+        statement.execute("EXECUTE SCRIPT DELETE_RLS_ROLE(" + roleToDelete + ")");
+        final ResultSet expectedResultSet = statement.executeQuery("SELECT * FROM " + test_table_projection_name);
+        final ResultSet actualResultSet = statement.executeQuery("SELECT * FROM " + test_table_name);
+        ScriptsSqlManager.dropTable(statement, EXA_ROLES_MAPPING);
+        ScriptsSqlManager.dropTable(statement, test_table_name);
+        ScriptsSqlManager.dropTable(statement, test_table_projection_name);
```

Comment:
Extract to a method `cleanUpTables(final String ... tableNames)`.

Reply:
done